recipe/meta.yaml

```diff
@@ -35,12 +32,228 @@ test:
 about:
   home: http://github.com/pahaz/sshtunnel
   license: MIT
-  license_file: LICENSE
```

Comment:
Why did you remove this?

Reply:
The pypi package has a license included in the source archive.  The only time you need to set license_file is when you're including a license in the recipe because the source archive doesn't have a license and license_file should be set to the path of the license file relative to the recipe directory.  This conda-forge feedstock repo doesn't contain a license file.